content/blog/api-console-for-data-services-cloud-console.md

```diff
@@ -0,0 +1,196 @@
+
+To access the API console, user must login into the [GreenLake Console,](https://common.cloud.hpe.com) deployed the DSCC to the intended region, and onboarded a storage array (Alletra, Nimble, or Primera) into the organization that is associated with the user account. The user must have the role that is required to perform the intended operation at the instance of DSCC where the storage is deployed. e.g. User must have volume management capability in the Data Ops Management to create a storage volume in US West. For more information about the role based access control, please take a look at the [HPE GreenLake User Guide](https://support.hpe.com/hpesc/public/docDisplay?docId=ccs-help_en_us)
+
+The **API** **Gateway** menu is available inside **Manage** menu. From **Greenlake Console** click on **Menu** to access this **Manage** Menu.
+
+![Access Menu ](/img/accesing-manage-menu-from-console.png "Menu in Cloud Console")
+
+![CCS Menu](/img/accesing-api-gateway-from-manage-menu.png "GreenLake Common Cloud Menu")
+
+The API Gateway provides the following operations:
+
+1. Create and manage API client applications association to obtain:
+
+   1. Instance ID of the DSCC at the particular region (Hexadecimals)
+   2. Client ID (Hexadecimals)
+   3. Client Secret (Hexadecimals)
+   4. URL to the HPE GreenLake Console API end-point (string)
+   5. URL to the HPE DSCC API end-point (string)
+2. Generate Access Token, change Client Secret, and delete Access Token.
+
+![API Gateway](/img/api-gateway-block.png "DSCC API Gateway")
+
+### Manage API Client Applications
+
+Each of the API client Applications Connection creates the relationship between the client and server (application). Click on the **Create Credentials** button to generate a client credential to generate the Access Token.
+
+![](/img/create-credentials-button.png)
+
+Inside the Create Credentials menu, click on the V button to show the pull down list and use the mouse to click on the desired application. i.e. For storage select the Data Services Cloud Console. The list also show region where the applications is deployed.
+
+![select the application](/img/select-the-desired-application.png "Choose application (DSCC)")
+
+After selecting the correct application enter the Credential Name (Please see HPE GreenLake Cloud Console user guide for supported characters.) Click the Create Credentials button to proceed with Client Credentials creation.
+
+![Create Credential no input yet](/img/create-credentials-menu.png "Generate Client Credentials 1st time")
+
+Note that to create the credentials for the OAuth, you have to provide the application and the name of this credential. Select the Data Services Cloud Console Application on the appropriate region where the console are deployed for this organization.
+
+Once the Create Credential is selected, the following information about the OAuth will be shown.
+
+![](/img/api-client-credential-created.png "Credentials Created Close")
```

Comment:
this image is a little blurry. Any chance you have one with a higher resolution?